testsuite/classlibrary/TestUnitTest.sc

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
I think these unit tests should go into `testsuite`, shouldn't they?

Reply:
uhm, aren't they? filename reads `testsuite/classlibrary/TestUnitTest.sc`

Reply:
I could swear they read something else :)